automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/azure/AzureRuntimeSpec.scala

```diff
@@ -172,12 +172,15 @@ class AzureRuntimeSpec
         )(implicitly, GeneratedLeonardoClient.runtimeInStateOrError(ClusterStatus.DELETED))
 
         _ <- loggerIO.info(
-          s"AzureRuntime: runtime ${workspaceId}/${runtimeName.asString} delete monitor result: $monitorDeleteResult"
+          s"AzureRuntimeSpec: runtime ${workspaceId}/${runtimeName.asString} delete monitor result: $monitorDeleteResult"
         )
         _ = monitorDeleteResult.getStatus() shouldBe ClusterStatus.DELETED
 
         diskAfterRuntimeDelete <- getDisk
-        _ = diskAfterRuntimeDelete.getStatus shouldBe DiskStatus.DELETED
+        _ <- loggerIO.info(
+          s"AzureRuntimeSpec: disk $workspaceId/${diskAfterRuntimeDelete.getName} delete monitor result: $diskAfterRuntimeDelete"
+        )
+        _ = diskAfterRuntimeDelete.getStatus should (be(DiskStatus.DELETED) or be(DiskStatus.DELETING))
```

Comment:
This should be fine, since occasionally disks take longer than VMs to delete on Azure